general.py

```diff
@@ -0,0 +1,74 @@
+def typecheck(arr):
+    check = [type(el) == int for el in arr]
```

Comment:
It's actually bad practice to do type checks like this in python instead you could import typing module:

def typecheck(arr: List[int]) -> bool:

This doesn't enforce types but it suggest to user what type is

Reply:
If you were to check types you should use:

if isinstance(my_number, int):